src/plugins/core/range.ts

```diff
@@ -373,6 +373,8 @@ export class RangeAdapter implements CommandHandler<CoreCommand> {
     if (prefixSheet) {
       if (rangeImpl.invalidSheetName) {
         sheetName = rangeImpl.invalidSheetName;
+      } else if (!this.getters.tryGetSheet(rangeImpl.sheetId)) {
+        sheetName = INCORRECT_RANGE_STRING;
```

Comment:
The root cause is that we don't adapt ranges that are in the clipboard, right?
(it doesn't harm that `getRangeString` is robust to missing sheets, but it may hide the root cause of a bug)

Reply:
Not sure but I don't think we want to adapt the ranges in the clipboard ? That's the whole discussion we already had 4 times about pasting things with the current state of the sheet VS the state at the time of pasting.

If we adapt the ranges inside the clipboard, that means that copy col A => delete col A => paste col A does not work

Reply:
Maybe we could put the chart Definiton instead of the abstract chart in the clipboard tho, that may solve the issues